week-9/Homework/mandatory/2-exercises/3-atm.js

```diff
@@ -12,14 +12,25 @@
  */
 
 class ATM {
-    // Add your code here
-
+  // Add your code here
+  constructor() {
+    this.balance = 100;
+  }
+  make_deposit(deposit) {
+    this.balance = this.balance + deposit;
+  }
+
+  make_withdrawl(withdraw) {
+    this.balance = this.balance - withdraw;
+  }
+  check_balance() {
+    console.log(this.balance);
+  }
 }
 
 let atm = new ATM(); // Create the ATM
 
 atm.make_deposit(200);
-atm.check_balance();
 atm.make_withdrawl(100);
-
-atm.make_withdrawl(500); // Your ATM should be able to handle this scenario
\ No newline at end of file
+atm.make_withdrawl(500); // Your ATM should be able to handle this scenario
```

Comment:
Your ATM allows people to withdraw 500 even though they don't have enough money in their balance.

It would be good to add error handling that will give the user a messaging saying they don't have enough money.